selectel/resource_selectel_mks_cluster_v1.go

```diff
@@ -45,6 +45,12 @@ func resourceMKSClusterV1() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					if strings.ToLower(old) == strings.ToLower(new) {
```

Comment:
Could be simplified: 
`return strings.ToLower(old) == strings.ToLower(new)`

Reply:
thanks!

Reply:
looks good, but `staticcheck` linter reports: 
`SA6005: should use strings.EqualFold(a, b) instead of strings.ToLower(a) == strings.ToLower(b)`